src/NovaBelongsToDepend.php

```diff
@@ -79,9 +80,12 @@ public function optionsResolve($callback)
         return $this;
     }
 
-    public function dependsOn($relationship)
+    public function dependsOn(string ...$classNames): NovaBelongsToDepend
     {
-        $this->dependsOn = Str::lower($relationship);
+        foreach ($classNames as &$value) {
```

Comment:
Continuing the effort to make this a non breaking change,  if `$classNames` has one entry, assign it to `dependsOn`, otherwise create and assign `$classNames`